src/writerly.gleam

```diff
@@ -624,6 +623,38 @@ fn tentative_parse_at_indent(
                   let assert True = suffix_indent == indent
 
                   case nonempty_suffix_diagnostic(suffix) {
+                    BackwardSlash(suffix) -> {
+                      let blame = blame
+                      let blamed_content = BlamedContent(blame, suffix)
+
+                      let #(more_blamed_contents, head_after_others) =
+                        fast_forward_past_other_lines_at_indent(
+                          indent,
+                          move_forward(head),
+                        )
+
+                      let tentative_blurb =
+                        TentativeBlurb(
+                          blame: blame,
+                          contents: list.prepend(
+                            more_blamed_contents,
+                            blamed_content,
+                          ),
+                        )
+
+                      let #(
+                        siblings,
+                        siblings_trailing_blank_lines,
+                        head_after_indent,
+                      ) = tentative_parse_at_indent(indent, head_after_others)
+
+                      #(
+                        list.prepend(siblings, tentative_blurb),
+                        siblings_trailing_blank_lines,
+                        head_after_indent,
+                      )
+                    }
+
```

Comment:
@vistuleB this is basically same as `Other(_)` (text blurb) case since I identified they both acts as same and difference is text blurb now has `annotation` with `\` identifier.

This whole case expression could be refactored into a function to keep it DRY for both cases (`BackwardSlash` and `Other`) but I thought future feature might change implementation so I refrain myself from doing it.